src/frontend/web/src/ratingpage/EffortBar.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
```

Comment:
If I understand correctly -  the effort bar is this component inside the card:
![image](https://user-images.githubusercontent.com/44426592/77845109-d633ea80-71b4-11ea-9363-e42eeab004a0.png)
I think it shouldn't be in this directory, but in one that contains global components (call it components) in the same hierarchy as this directory. 
The reason is that it will probably be used in many places in the website - and it doesn't belong specifically to the rating page.